Makefile

```diff
@@ -39,6 +38,10 @@ checkcoverage:
 
 .PHONY: lint
 lint:
+	@if ! $(shell go env GOPATH)/bin/golangci-lint --version &>/dev/null; then \
+		echo "Installing linters..."; \
+		curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin v1.22.2; \
```

Comment:
This installs the linter successfully, but running the linter fails because `@golangci-lint run --deadline=10m --tests --disable-all \` cannot find the binary.

We should use `$(shell go env GOPATH)/bin/golangci-lint` when calling the linter so that the path can be consistently resolved. Also, I think it would make sense to have a variable for `$(shell go env GOPATH)/bin/`.

Reply:
this commit really seems out of place. This PR was intended just to move mixer to go mod, not change the behavior of make targets.

Reply:
@jwakre good catch.. I had my PATH set to my home dir... Hence was working for me.. Will make the change in another PR..

Reply:
Removing the lint commit and merging..